piton/tools/src/sims/sims,2.0

```diff
@@ -2720,7 +2798,7 @@ sub parse_args
   push (@{$opt{config_rtl}}, "ORAM_ON") if ($opt{oram}) ;
   push (@{$opt{sim_run_args}}, "+oram") if ($opt{oram}) ;
 
-  if ($opt{msm_build} or $opt{icv_build} or $opt{vlt_build}) {
+  if ($opt{msm_build} or $opt{icv_build} or $opt{vlt_build} or $opt{riv_build}) {
```

Comment:
Only remaining thing I think was whether you needed this - the other simulators (at least icarus and verilator) don't support these PLI-based core modifying/monitoring parts of the non-synthesisable code that we inherited from OpenSPARC.

If you remove `riv_build` from this if statement, do the tests still build and run? If so, you should be good to leave it out.